platform/integrations/karpenter.mdx

```diff
@@ -0,0 +1,482 @@
+
+<Tabs
+  defaultValue="EKS"
+  values={[
+    { label: "Provision NodePool for EKS", value: "EKS" },
+    { label: "Provision NodePool for AKS", value: "AKS" },
+  ]}
+>
+  <TabItem value="EKS">
+
+    ```bash
+    cat <<EOF | envsubst | kubectl apply -f -
+    apiVersion: karpenter.sh/v1beta1
+    kind: NodePool
+    metadata:
+      name: <name-of-your-choosing>
+    spec:
+      template:
+        spec:
+          taints:
+          - key: <project-name>
+            value: "true"
+            effect: NoSchedule
+          requirements:
+            - key: kubernetes.io/arch
+              operator: In
+              values: ["amd64"]
+            - key: kubernetes.io/os
+              operator: In
+              values: ["linux"]
+            - key: karpenter.sh/capacity-type
+              operator: In
+              values: ["spot"]
+            - key: karpenter.k8s.aws/instance-category
+              operator: In
+              values: ["c", "m", "r"]
+            - key: karpenter.k8s.aws/instance-generation
+              operator: Gt
+              values: ["2"]
+          nodeClassRef:
+            apiVersion: karpenter.k8s.aws/v1beta1
+            kind: EC2NodeClass
+            name: <node-class-name>
+      limits:
+        cpu: 100
+      disruption:
+        consolidationPolicy: WhenUnderutilized
+        expireAfter: Never
+EOF
+```
+
+  </TabItem>
+  <TabItem value="AKS">
+
+    ```bash
+    cat <<EOF | envsubst | kubectl apply -f -
+    apiVersion: karpenter.sh/v1beta1
+    kind: NodePool
+    metadata:
+      name: <name-of-your-choosing>
+    spec:
+      template:
+        spec:
+          taints:
+          - key: <project-name>
+            value: "true"
+            effect: NoSchedule
+          requirements:
+            - key: kubernetes.io/arch
+              operator: In
+              values: ["amd64"]
+            - key: kubernetes.io/os
+              operator: In
+              values: ["linux"]
+            - key: karpenter.sh/capacity-type
+              operator: In
+              values: ["on-demand"]
+            - key: karpenter.azure.com/sku-family
+              operator: In
+              values: [D]
+          nodeClassRef:
+            apiVersion: karpenter.k8s.aws/v1beta1
+            kind: AKSNodeClass
+            name: <node-class-name>
+      limits:
+        cpu: 100
+      disruption:
+        consolidationPolicy: WhenUnderutilized
+        expireAfter: Never
+EOF
+    ```
+
+  </TabItem>
+</Tabs>
+
+4. Create a new project that only allows template(s) with set tolerations. Go to project dropdown and create a new project. In the Allowed Templates section, remove the All Templates option. Select the template(s) that include tolerations for the node pool created in step 3. Now, all pods created in virtual clusters belonging to this project will deploy to the designated node pool(s).
+
+### Target Node Pool Based on vCluster Pod Labels
```

Comment:
This heading size looks wrong based on the table of contents on the right hand side of the preview.